app/components/primer/alpha/overlay.pcss

```diff
@@ -17,14 +19,496 @@ primer-overlay[popover='manual' i] {
   inset-inline-end: 0;
   inset-block-start: 0;
   inset-block-end: 0;
-}
+} */
 
 /* stylelint-disable selector-class-pattern */
-primer-overlay[popover='' i].\:open,
+
+/* primer-overlay[popover='' i].\:open,
 primer-overlay[popover='auto' i].\:open,
 primer-overlay[popover='manual' i].\:open {
   display: block;
   position: fixed;
   z-index: 2147483647;
-}
+} */
+
 /* stylelint-enable selector-class-pattern */
```

Comment:
I only commented out these `primer-overlay` styles. It seems to be "test styles", but I assume they can be removed again at some point?

Before | After
--- | ---
![Screen Shot 2022-11-29 at 17 56 52](https://user-images.githubusercontent.com/378023/204511347-249d8ea4-0464-4922-914b-ba13c0745d25.png) | ![Screen Shot 2022-11-29 at 17 57 32](https://user-images.githubusercontent.com/378023/204510650-9782d3bf-e82a-492d-ad61-675cd11c0f9a.png)